src/main/java/it/pagopa/dss/Utility.java

```diff
@@ -0,0 +1,94 @@
+package it.pagopa.dss;
+
+import eu.europa.esig.dss.model.DSSDocument;
+import eu.europa.esig.dss.pades.exception.InvalidPasswordException;
+import eu.europa.esig.dss.pades.validation.ByteRange;
+import eu.europa.esig.dss.pdf.PAdESConstants;
+import eu.europa.esig.dss.pdf.PdfArray;
+import eu.europa.esig.dss.pdf.PdfDict;
+import eu.europa.esig.dss.pdf.pdfbox.PdfBoxDocumentReader;
+import eu.europa.esig.dss.utils.Utils;
+import it.pagopa.dss.exception.SignatureServiceException;
+import java.io.IOException;
+import java.util.List;
+import org.apache.pdfbox.cos.COSDictionary;
+import org.apache.pdfbox.cos.COSName;
+import org.apache.pdfbox.cos.COSObject;
+import org.apache.pdfbox.pdmodel.PDDocument;
+import org.apache.pdfbox.pdmodel.interactive.form.PDSignatureField;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+final class Utility {
+
+  private Utility() {}
+
+  private static final Logger LOG = LoggerFactory.getLogger(Utility.class);
+
+  static ByteRange getByteRange(DSSDocument documentToSign, String signatureFieldId)
```

Comment:
can you add some comments clarifying what's happening here?

Reply:
surely. I will try to clarify as best I can!